Doppler.Jobs.Server/Startup.cs

```diff
@@ -24,13 +34,66 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddSingleton(typeof(ILoggerAdapter<>), typeof(LoggerAdapter<>));
```

Comment:
Why do you need this adapter?

Reply:
For testing purpose, let me explain in this [link ](https://chrissainty.com/unit-testing-ilogger-in-aspnet-core/) maybe you have a better solution
For example https://github.com/MakingSense/doppler-jobs/pull/1/files#diff-5512938629fd8e6623b4aae53c1332c8R38 
I want test the text for logger `_loggerMock.Verify(x => x.LogInformation(
                "Getting currency per each code."), Times.Once);`, with this adapter I can test the text log exactly, I used the solution two of the link for this test

Reply:
But `ILogger` is already an interface, you can mock `ILogger` directly, without an adapter.

Reply:
Yes but if I use ILogger interface I have to test with this `Verify ` sentence
` _mockLogger.Verify(x => x.Log(LogLevel.Error, It.IsAny<EventId>(), It.IsAny<FormattedLogValues>(), It.IsAny<Exception>(), It.IsAny<Func<object, Exception, string>>()), Times.Once);`
With this sentence, I cannot check the text logger